File: packages/dotnet-sdk/src/Blindfold.Sdk/Models/DetokenizeResponse.cs
using System.Text.Json.Serialization;

namespace Blindfold.Sdk.Models;

public class DetokenizeResponse
{
    [JsonPropertyName("text")]
    public string Text { get; set; } = "";

    [JsonPropertyName("replacements_made")]
    public int ReplacementsMade { get; set; }
}

File: packages/java-sdk/src/main/java/dev/blindfold/sdk/models/DetokenizeResponse.java
package dev.blindfold.sdk.models;

import com.google.gson.annotations.SerializedName;

public class DetokenizeResponse {
    private String text;

    @SerializedName("replacements_made")
    private int replacementsMade;

    public DetokenizeResponse() {}

    public String getText() { return text; }
    public int getReplacementsMade() { return replacementsMade; }
}
